docs/operations/replication.md

```diff
@@ -0,0 +1,225 @@
+    kubectl --namespace=storageos get pod -ocustom-columns=_:.metadata.name --no-headers -lapp=storageos-cli
+
+    storageos-cli-77885d6d8b-mgbr8
+    ```
+
+1.  With the Ondat CLI now deployed, you can check the count and location of the master and replica volumes created for `pvc-replicated`.
+
+    ```bash
+    # Get the volumes in the "default" namespace using the Ondat CLI.
+    kubectl --namespace=storageos exec storageos-cli-77885d6d8b-mgbr8 -- storageos get volumes --namespace=default
+
+    NAMESPACE  NAME                                      SIZE     LOCATION                     ATTACHED ON  REPLICAS  AGE
+    default    pvc-7fed5a56-42b2-4fe3-bcab-e31c97931b8c  5.0 GiB  demo-worker-node-1 (online)               1/1       8 minutes ago
+
+    # Describe the "pvc-7fed5a56-42b2-4fe3-bcab-e31c97931b8c" volume.
+    kubectl --namespace=storageos exec storageos-cli-77885d6d8b-mgbr8 -- storageos describe volume pvc-7fed5a56-42b2-4fe3-bcab-e31c97931b8c --namespace=default
+
+    ID                  495eb6a5-28de-4593-b6bc-d91094b52326
+    Name                pvc-7fed5a56-42b2-4fe3-bcab-e31c97931b8c
+    Description
+    AttachedOn
+    Attachment Type     detached
+    NFS
+      Service Endpoint
+      Exports:
+    Namespace           default (b4c020a6-6c54-40cd-a502-9ea4f4b68a9c)
+    Labels              csi.storage.k8s.io/pv/name=pvc-7fed5a56-42b2-4fe3-bcab-e31c97931b8c,
+                        csi.storage.k8s.io/pvc/name=pvc-replicated,
+                        csi.storage.k8s.io/pvc/namespace=default,
+                        storageos.com/nocompress=true,
+                        storageos.com/replicas=1
+    Filesystem          ext4
+    Size                5.0 GiB (5368709120 bytes)
+    Version             Mg
+    Created at          2022-07-25T16:47:51Z (10 minutes ago)
+    Updated at          2022-07-25T16:47:53Z (10 minutes ago)
+
+    Master:
+      ID                24a56198-1648-46af-ac25-71d02ecba31c
+      Node              demo-worker-node-1 (d7a745ff-242a-48f4-a6bf-4f191c14a237)
+      Health            online
+
+    Replicas:
+      ID                b7f565c6-860b-4458-aeb2-c2f623da77af
+      Node              demo-worker-node-4 (cad83b41-89b7-4520-9b82-632f31d94814)
+      Health            ready
+      Promotable        true
+    ```
+
+    > 💡 As demonstrated in the output above, notice the number of master replica volumes created and which node they are located on. If we created a volume without a replica count defined in *Step 1* - only the master volume would be provisioned. 
+
+### Example - Enable Volume Replication Through a `StorageClass` Definition
+
+The following guidance below will demonstrates how to use Ondat's Volume Replication  through a  `StorageClass`  (PVC) definition.
+-   The instructions will enable volume replication through a custom `StorageClass` and use the following parameter »  `storageos.com/replicas=2` - which will create `1` master volume and `2` replica volumes respectively.
```

Comment:
Is `»` meant to be there?

Reply:
yes - I was use it point to specific items, but I am happy to remove it